### ISSUE TITLE
Update npm package

### ISSUE THREAD

**Issue by DonsWayo** [erikc5000/nest-mongodb#3]
I see that you updated the dependencies, can update the npm package to run with latest dependencies please? ;)

**Comment by erikc5000** [comment]
The updates were only to the dev dependencies. I try to update to those with some regularity to make sure everything will work with the latest versions, but ultimately, you have control over the nestjs, mongodb, rxjs, and reflect-metadata versions that are actually used since those are defined as peer dependencies. If you want to update them, you should be able to do that in your project that's using nest-mongodb.

So basically, there's no new release to npm since there haven't been any meaningful changes for a consumer of the library. Make sense?

**Comment by DonsWayo** [comment]
Totally make sense, thanks for the aclaration.

PD: Thanks for the library!
